.env

```diff
@@ -1,8 +1,8 @@
 SASS_PATH=node_modules:src/assets/styles
-REACT_APP_SENTRY_DSN=
-REACT_APP_SENTRY_ENVIRONMENT=
-REACT_APP_SERVICE_MAP_URI=https://palvelukartta.hel.fi/unit/
-REACT_APP_API_URI=https://venepaikka-federation.test.kuva.hel.ninja/
+REACT_APP_API_URL=https://venepaikka-federation.test.kuva.hel.ninja/
+REACT_APP_SERVICE_MAP_URL=https://palvelukartta.hel.fi/unit/
+REACT_APP_SENTRY_DSN=https://id1@sentry.io/id2
```

Comment:
Could put the project's sentry DSN here `https://5db8799439aa4c7dbbd33878649ba4c9@sentry.hel.ninja/48`?

Reply:
Added